scripts/configure.js

```diff
@@ -136,6 +143,20 @@ function lldbVersionToBranch(version) {
 
 // On Mac the lldb version string doesn't match the original lldb versions.
 function getDarwinRelease() {
+  try {
+    const version = child_process.execFileSync('llvm-config', ['--version']).toString().trim();
+    const prefix = child_process.execFileSync('llvm-config', ['--prefix']).toString().trim();
+    const options = JSON.stringify({
+      variables: { "lldb_build_dir%": prefix }
+    }, null, 2);
+    fs.writeFileSync('options.gypi', Buffer.from(options), 'utf-8');
```

Comment:
Using `Buffer.from()` means this script will fail for people stuck on node.js versions < v4.5.0.  It also doesn't look necessary to me or am I missing something?

Reply:
You're right, we don't need to transform this into a buffer